### ISSUE TITLE
Scrolling through Kit in Keyboard view should use Horizontal knob instead of Vertical knob.

### ISSUE THREAD

**Issue by ok-reza** [alter-alter/DelugeFirmware#5]
When scrolling position in Keyboard view, the user should use the <> knob instead of the vertical knob. A pad will move to across 4 times before moving up or down a row 1 time, so the action feels inherently horizontal. After leaving Keyboard view, it would then resume being vertical scroll in the sequencer view.

Minor and subjective, but might make the experience a little easier to understand.

**Comment by found-sounds** [comment]
> When scrolling position in Keyboard view, the user should use the <> knob instead of the vertical knob. A pad will move to across 4 times before moving up or down a row 1 time, so the action feels inherently horizontal. After leaving Keyboard view, it would then resume being vertical scroll in the sequencer view.
> 
> 
> 
> Minor and subjective, but might make the experience a little easier to understand.

It doesn't make sense to change the behavior of scrolling from one knob to the other when on different pages of drum kits. This would personally confuse me as well as make things more complicated on the back end

**Comment by ok-reza** [comment]
You are right! The behavior of scrolling from one knob should be the same across clip view and keyboard view.

So perhaps the solution is that, with the Vertical knob, the large drum pads should move up and down in Drum Keyboard View by 4-pad-rows instead of moving 4-pad-columns left and right. This means two turns would scroll to a different group of 8 large pads.

In reality, having both Horizontal and Vertical scroll would be ideal in Drum Keyboard View so you can have drums placed wherever you like. This would mimic Synth Keyboard View and probably be the most expected behavior.
